### DISCUSSION TITLE
I can't re-verify my academic affiliation

### DISCUSSION THREAD

**Discussion by yangsenlin01** [community/community#65868] [GitHub Education]
### Hi everyone,

我在2022年8月第一次申请通过GitHub benefits。现在它提示我需要重新验证学生身份。但是当我重新申请后（我现在仍然是一名学生），得到的提示是'You were last verified as a student on . It is not necessary for you to re verify at this time.'

I applied for GitHub benefits for the first time in August 2022. Now it prompts me to revalidate my student identity. But when I reapplied (I am still a student), the prompt I received was' You were last verified as a student on. ' It is not necessary for you to re verify at this time. '

**Comment by STARK-775178** [comment]
好兄弟，解决了吗 8D
